### DISCUSSION TITLE
Climate entity not displaying anything.

### DISCUSSION THREAD

**Discussion by BigHomie90** [Blackymas/NSPanel_HA_Blueprint#1666] [Q&A]
After the latest update (4.2.2) climate entity based on smartir custom component not displaying anything in ns panel. Does anyone else face the same issue?

**Comment by edwardtfn** [comment]
We have just release v4.2.4 with some fixes around climate entities. If you still having this issue after updating to v4.2.4, could you please report it as a bug in the [Issues](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues) area?
